src/com/base/game/character/body/Arm.java

```diff
@@ -184,6 +184,25 @@ public String setType(GameCharacter owner, ArmType type) {
 							+ "</p>");
 				}
 				break;
+			case COW_MORPH:
+				if (owner.isPlayer()) {
+					UtilText.transformationContentSB.append(
+								" Within a matter of moments, a layer of [pc.armFullDescription] has quickly grown over them, and, looking down,"
+										+ " you see your hair growing over the backs of your new hands as tough, hoof-like nails push out in place of regular, human-like ones."
+								+ " Despite their appearance, you're relieved to discover that your hands have lost none of their dexterity."
+								+ " As the transformation comes to an end, you see that at your upper-biceps, your new hair smoothly transitions into the [pc.skin] that's covering the rest of your body.</br>"
+								+ "You're left with anthropomorphic, [style.boldCowMorph(cow-like arms and hands)], which are covered in [pc.armFullDescription]."
+							+ "</p>");
+				} else {
+					UtilText.transformationContentSB.append(
+								" Within a matter of moments, a layer of [pc.armFullDescription] has quickly grown over them, and, looking down,"
```

Comment:
pc.armFullDescription

Reply:
It's Innoxia's code across all arms

Reply:
@Innoxia then this is a bug that NEEDS fixing

Reply:
Only for horses is it bugged

Reply:
I think I fixed that :s